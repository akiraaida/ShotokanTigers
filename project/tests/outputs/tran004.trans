10 Ashe Ace             00000 00000.00 S 
02 Ashe Ace             12348 01000.00   
02 Matt Cow             12346 01000.00   
01 Ashe Ace             12348 00000.10   
00                      00000 00000.00   
