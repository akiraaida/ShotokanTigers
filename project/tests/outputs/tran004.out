Please enter your session type:
Please enter a login name:
Please enter a command user:
Enter your account number:
Enter account number to transfer to:
Enter amount:
Success, 1000.00 was transferred to 12346.
Please enter a command: 
